grammar.js

```diff
@@ -75,6 +75,7 @@ module.exports = grammar({
     keyword_cross: _ => make_keyword("cross"),
     keyword_join: _ => make_keyword("join"),
     keyword_lateral: _ => make_keyword("lateral"),
+    keyword_natural: _ => make_keyword("natural"),
```

Comment:
this is already part of #196 right?

Reply:
Yes I used my previous MR to avoid conflicts.